src/bootstrap.php

```diff
@@ -156,14 +142,38 @@
         });
 
     $collections = $app['elife.api_sdk']->collections()
-        ->containing($article->getIdentifier())
+        ->containing(Identifier::article($id))
         ->slice(0, 100);
 
-    $podcastEpisodeChapters = $podcastEpisodes
-        ->reduce(function (Sequence $chapters, PodcastEpisode $episode) use ($article) {
+    $mostRecent = $app['elife.api_sdk']->search()
+        ->forType('research-advance', 'research-article', 'scientific-correspondence', 'short-report', 'tools-resources', 'replication-study')
+        ->sortBy('date')
+        ->slice(0, 5);
+
+    $mostRecentWithSubject = new PromiseSequence($article
+        ->then(function (ArticleHistory $history) use ($app) {
+            $article = $history->getVersions()[0];
+
+            if ($article->getSubjects()->isEmpty()) {
+                return new EmptySequence();
+            }
+
+            $subject = $article->getSubjects()[0];
+
+            return $app['elife.api_sdk']->search()
+                ->forType('correction', 'editorial', 'feature', 'insight', 'research-advance', 'research-article', 'retraction', 'registered-report', 'replication-study', 'scientific-correspondence', 'short-report', 'tools-resources')
+                ->sortBy('date')
+                ->forSubject($subject->getId())
+                ->slice(0, 5);
```

Comment:
To reduce processing time (on both ends). The majority of the time the most-recent-but-not-already-listed article is the first one.